VogtPayroll2/EmployeeHourly1.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    class EmployeeHourly : Employee
+    {
+        public EmployeeHourly(string aName, int aHoursWorked, decimal aPayRate)
+            : base(aName, aHoursWorked, aPayRate)
+        {
+            this.name = aName;
+            this.hoursWorked = aHoursWorked;
+            this.payRate = aPayRate;
+        }
+
+        public override decimal GetGrossPay()
+        {
+            decimal regularPay = default;
+            decimal overTime;
```

Comment:
Gross pay should not include the overtime pay. It wasn't obvious to me at first, but this is what sold me:
```
To compute the overtime pay for a salary based employee, first find the hourly rate by dividing the gross pay by 40, and then compute overtime pay.
```

Overtime should never be calculated off of itself. Gross must be the normal "agreed" upon 40/hr wk price.

I could be wrong but it would be weird 🤔

Reply:
I think this should just be `return hoursWorked * PayRate();`